src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
I would prefer to drop the `-rpath`. I think we should just assume the program will ensure that it is on the path somewhere (since we already know that it is able to load libLLVM).

Does this work on macOS? I don't see any equivalent command line flag, so it seems that the header file appraoch may be necessary.